docs/guides/monorepo.mdx

```diff
@@ -32,6 +32,35 @@ dev_dependencies:
   widgetbook_generator: ^{{ versions.generator }}
 ```
 
+#### Melos
+
+If you run into dependency issues when using [Melos](https://melos.invertase.dev/) you might need to change how you bootstrap the dependencies.
+
+```yaml title="widgetbook/pubspec.yaml"
+name: widgetbook_workspace
+
+dependencies:
+  widgetbook_annotation: ^{{ versions.annotation }}
+  widgetbook: ^{{ versions.widgetbook }}
+  my_design_system: v1.0.0
+  my_app: v1.0.0
+
+dev_dependencies:
+  build_runner:
+  widgetbook_generator: ^{{ versions.generator }}
+```
+
+```yaml title="melos.yaml"
+name: my_project
+
+packages:
+  - apps/**
+  - packages/**
+  - widgetbook/
```

Comment:
**suggestion:** Let's highlight the `- widgetbook/` entry to grab readers attention.

```suggestion
  - widgetbook/ // [!code ++]
```